app/jobs/generate_master_skill_pdf_job.rb

```diff
@@ -0,0 +1,9 @@
+class GenerateMasterSkillPdfJob < ApplicationJob
+  queue_as :default
+
+  def perform(occupation_standard_id)
+    os = OccupationStandard.find(occupation_standard_id)
+    pdf = ::MasterSkill.new(os)
+    os.pdf.attach(io: StringIO.new(pdf.render), filename: "master_skill_#{os.id}.pdf")
```

Comment:
For the filename, i think using a name the user would recognize -- a parameterized `occupation_standard#title` for ex -- and our ID is preferable here. 

They're not going to know what our IDs mean, but we can leave those in for ourselves and name clashes.